language/stdlib/modules/0L/Errors.move

```diff
@@ -16,8 +16,10 @@ address 0x1 {
 /// >TODO: determine what kind of stability guarantees we give about reasons/associated module.
 module Errors {
     /// A function to create an error from from a category and a reason.
-    fun make(category: u8, reason: u64): u64 {
```

Comment:
@sm86 I wonder if you can move the module prefix here, instead of hand coding in each of the modules. I.e. change make() signature to include the module code.

Reply:
I considered it, but Move does not support strings, so prefix is not straight forward. So I choose to go with u64 codes

Reply:
What I mean is we are using a number code for each module. Why not pass that number here, and concatenate, instead of doing manually?

Reply:
I'm passing the concatenated value - it has module number + function number